### PR TITLE
HYDRA-859 : Fix MAYAUSD_MOD_PATH not being part of the downloaded May…

### PR DESCRIPTION
**Pull request by lanierd-adsk**
…aUsd cuts.

### CODE REVIEW

File: cmake/test.cmake [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
-    # Get the current value of the environment variable
-    set(CURRENT_MAYA_MODULE_PATH $ENV{MAYA_MODULE_PATH})
-    # Append the new path to the current value
-    if(IS_MACOSX OR IS_LINUX) 
-        #Linux and OSX
-        set(MAYA_MODULE_PATH "${CURRENT_MAYA_MODULE_PATH}:${MAYAUSD_MOD_PATH}")
-    else()
-        #Windows
-        set(MAYA_MODULE_PATH "${CURRENT_MAYA_MODULE_PATH}\;${MAYAUSD_MOD_PATH}")
+    if (MAYAUSD_MOD_PATH)
```
Comment (lanierd-adsk) [inline cmake/test.cmake:4]: The only change here is the testing of MAYAUSD_MOD_PATH. Before, this was not done as MayaUsd_FOUND would have been false if MAYAUSD_MOD_PATH was not found.

**Comment by lanierd-adsk** [comment]
A test is failing, I am checking why and fixing it.

**Comment by lanierd-adsk** [comment]
FYI : @debloip-adsk @vlasovi @lilike-adsk @roopavr-adsk

File: cmake/modules/FindMayaUsd.cmake [hunk 2]
```diff
@@ -50,7 +50,8 @@ find_package_handle_standard_args(MayaUsd
     REQUIRED_VARS
         MAYAUSD_INCLUDE_DIR
         MAYAUSDAPI_LIBRARY
-        MAYAUSD_MOD_PATH
+        #MAYAUSD_MOD_PATH is not part of MayaUsd cuts downloaded so remove it from the requirements
```
Comment (ppt-adsk) [inline cmake/modules/FindMayaUsd.cmake:53]: This won't be meaningful to external developers.  I would just remove this comment.

**Review by ppt-adsk — APPROVED** [review]

File: cmake/test.cmake (hunk 1 above)
Comment (seando-adsk) [reply cmake/test.cmake:4]: You could have also done `if(MayaUsd_FOUND AND MAYAUSD_MOD_PATH)`

File: cmake/modules/FindMayaUsd.cmake (hunk 2 above)
Comment (seando-adsk) [reply cmake/modules/FindMayaUsd.cmake:53]: I would maybe put a comment on line 11 (where this var is described) saying its optional

**Review by seando-adsk — APPROVED** [review]

**Comment by lanierd-adsk** [comment]
This PR needed HYDRA-814 to be merged in to pass the tests. A new PR has been created with that change :
https://github.com/Autodesk/maya-hydra/pull/68
So this PR is being closed.

**Pull request closed without merging**